database/src/main/resources/changesets/metadata_journal_subsecond_timestamp.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet id="metadata_journal_subsecond_timestamp" author="mcovarr">
+        <modifyDataType
+                columnName="METADATA_TIMESTAMP"
+                newDataType="DATETIME(6)"
+                tableName="METADATA_JOURNAL"/>
+    </changeSet>
+
+    <changeSet id="metadata_journal_timestamp_not_null" author="mcovarr">
+        <addNotNullConstraint
+                tableName="METADATA_JOURNAL"
+                columnName="METADATA_TIMESTAMP"
+                columnDataType="DATETIME(6)"/>
```

Comment:
Are both of these really necessary? Presumably if the second query requires the data type, it allows it to be modified? (otherwise what's the point!?)